gwtlib/worktrees.py

```
@@ -169,58 +293,345 @@ def remove_worktree(branch_name: str, git_dir: str) -> None:
+                )
+                deleted_remote = True
+            else:
+                # Check if it's already deleted (not a real error)
+                if "remote ref does not exist" in error_msg.lower():
+                    print(
+                        f"Remote branch '{remote_name}/{branch_name}' already deleted",
+                        file=sys.stderr,
+                    )
+                    deleted_remote = True
+                else:
+                    raise RuntimeError(f"Failed to delete remote branch: {error_msg}")
```

Ruff (0.15.10)

[warning] Abstract `raise` to an inner function

(TRY301)